tests/dragonfly/cluster_test.py

```diff
@@ -1840,16 +1858,45 @@ async def test_cluster_migration_huge_container(df_factory: DflyInstanceFactory)
     await push_config(json.dumps(generate_config(nodes)), [node.admin_client for node in nodes])
 
     logging.debug("Generating huge containers")
-    seeder = StaticSeeder(
-        key_target=10,
-        data_size=10_000_000,
-        collection_size=10_000,
-        variance=1,
-        samples=1,
-        types=["LIST", "HASH", "SET", "ZSET", "STRING"],
-    )
-    await seeder.run(nodes[0].client)
-    source_data = await StaticSeeder.capture(nodes[0].client)
+    # Insert data to containers with a gaussian distribution: some will be small and other big
+    stop = False
+
+    async def insert_data(client):
```

Comment:
Kostas is working on seeder improvement for containers with different size
I prefer to have something generic and this logic inside test